spec/forms/provider_interface/course_wizard_spec.rb

```diff
@@ -167,6 +178,24 @@
           expect(wizard.next_step).to eq(:locations)
         end
       end
+
+      context 'when there is only one location available' do
+        before do
+          allow(query_service).to receive(:available_course_options).and_return([create(:course_option)])
```

Comment:
Could this be `build_stubbed(:course_option)` instead of `create` ?

Reply:
👍